backend/src/main/java/com/bakdata/conquery/models/config/LocaleConfig.java

```diff
@@ -64,20 +76,57 @@
 	private List<ListFormat> listFormats = List.of(
 			new ListFormat("", ", ", ""),
 			new ListFormat("{", ",", "}"),
-			new ListFormat("[", ",", "]"));
+			new ListFormat("[", ",", "]")
+	);
 
 	/**
 	 * Container to describe the format of a list
 	 */
 	@Data
-	@AllArgsConstructor
+	@RequiredArgsConstructor
+	@Slf4j
 	public static class ListFormat {
-		@NonNull @Max(1)
-		String start;
-		@NonNull @Min(1)
-		String separator;
-		@NonNull @Max(1)
-		String end;
+		@NonNull
+		@Size(min = 0, max = 1)
+		private final String start;
+		@NonNull
+		@Size(min = 1)
+		private final String separator;
+
+		@NonNull
+		@Size(min = 0, max = 1)
+		private final String end;
+
+		/**
+		 * Manually parse value as {@link CDateSet} using the reader for {@link CDateRange}.
+		 */
+		public CDateSet parse(String value, DateReader reader) {
+
+			// Assert that value starts/ends properly
+			if (!value.startsWith(getStart())) {
+				throw new ParsingException(String.format("Expected Start `%s`.", getStart()));
+			}
+
+			if (!value.endsWith(getEnd())) {
+				throw new ParsingException(String.format("Expected End `%s`.", getEnd()));
+			}
+
+			// We strip start and end
+			value = value.substring(getStart().length(), value.length() - getEnd().length());
+
+			final StringTokenizer tokenizer = new StringTokenizer(value, getSeparator().trim());
```

Comment:
kann der Separator nicht ein Leerzeichen sein, wenn ja, könnte der Aufruf der Trim Methode problematisch sein

Reply:
Du hast recht, das würde ich noch mit @thoniTUB besprechen (in einem Kommentar weiter oben im Code erwähnt er, dass das Leerzeichen in der Ausgabe optional ist, aber eigentlich ist das nicht wichtig.) Ich hatte das so umgesetzt, weil der StringTokenizer leider so implementiert ist, dass er den Separator-String als Char Array betrachtet. Ich hab es jetzt umgeschrieben mit String.split was eigentlich keinen großen Nachteil hat

Reply:
den code gibt es nicht mehr. Kann resolved werden oder?